data_pipelines/resources/__init__.py

```diff
@@ -22,4 +26,11 @@
     ),
     "zarr_io_manager": ZarrIOManager(base_path=settings.base_data_path),
     "parquet_io_manager": ParquetIOManager(base_path=settings.base_data_path),
+    "client": CDSClient(user_id=EnvVar("CDS_USER_ID"), api_key=EnvVar("CDS_API_KEY")),
```

Comment:
Maybe rename the key to `"cds_client"` or something similar, as there will probably be more client resources.